### DISCUSSION TITLE
Can we delete the framework or it can be only unlinked with the channels and root orgs?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#253] [Documentation] [answered]
If I've somehow created the wrong framework and linked it to a tenant.
I can unlink it from the tenant but how am I supposed to delete the framework from our instance?

cc: @rajeevsathish @reshmi-nair @Krishnaj20

**Comment by chitranshu-keshav** [comment]
@nidhikachhawa24 is your issue resolved? If yes then can you please update the resolution and mark this thread as answered? If not then please follow up with relevant details
cc @Krishnaj20

**Comment by vinod303** [comment, marked as answer]
@nidhikachhawa24  you should be able to delete the framework using the  framework retire api 

`curl --location --request DELETE '<URL>/action/framework/v3/retire/<FrameworName>' \`
